docs/features/lists.md

```diff
@@ -30,4 +64,39 @@ There are three ways to add a list:
 
 Once your list is created, you'll be able to see an overview of all the contributors in your list, a graph of their activity, and a detailed breakdown of the contributors.
 
+## How to Create a List
+
+After logging in, navigate to your Lists Page.
+
+
+## Creating a New List
+
+1. Click the "Lists" tab on the Insights hub to create a new list. You will be redirected to a page where you can create your new list.
+2. Click on the "New List" button to start a new list.
+3. Give your list a name.
+4. Choose your page visibility. You can choose to make your list public or private. By default, it is set to private.
+5. Add contributors or maintainers to your list by searching for their GitHub username, syncing your GitHub Team, or importing your GitHub Following.
+
+![Here](https://docs.opensauced.pizza/assets/images/team-sync-09e7e2ab7b54fae335f5b4c40cce94b1.gif).
+
+## Using Your List
+
+Once you have created your list, you will be redirected to your List Page. Here, you will see a dashboard with an overview of the contributions in your list.
+
+There are three tabs that provide you with more information on these repositories:
+
+## Overview
+
+The Overview tab gives you a high-level view of the contributors in your list, including the total number of commits in the last 30 days and the types of contributors on your list: Active, New, and Alumni.
+
+## Activity
+
+The Activity tab gives you a graph view with more detailed information on each contributor, including the type of activity, repositories they've contributed to, and how they compare to each other.
+
```

Comment:
There's also the Highlights tab that shows any Highlights on OpenSauced that have been made by the contributors on the list.

Reply:
I don't see the section on Highlights here.